server/helpers/tag_helpers.py

```diff
@@ -0,0 +1,50 @@
+import calendar
+from typing import Dict, List
+
+from server.models import Tag
+
+
+def build_categorised_tags(tags: List[Tag]) -> Dict:
+
+    categorised_tags = {}
+
+    for tag in tags:
+
+        category = tag.category.name
+        if category in categorised_tags.keys():
+            categorised_tags[category].append(tag.name)
+        else:
+            categorised_tags[category] = [tag.name]
+
+    return categorised_tags
+
+
+def tags_from_exif_data(exif_data: Dict) -> Dict:
+
+    full_date = exif_data['DateTimeOriginal']
+    date = full_date.split()[0]
```

Comment:
datetime and format has all the tools you need to get all the date/time values you want

Reply:
It does and I was being lazy, will resolve good catch